charts/pxc-db/README.md

```diff
@@ -81,9 +81,9 @@ The chart can be customized using the following configurable parameters:
 | `haproxy.tolerations`                   | List of node taints to tolerate for HAProxy Pods                       | `[]`                                      |
 | `haproxy.gracePeriod`                   | Allowed time for graceful shutdown                       | `600`                                      |
 | `haproxy.podDisruptionBudget.maxUnavailable` | Instruct Kubernetes about the failed pods allowed quantity           | `1`                                      |
-| `haproxy.persistence.enabled` | Requests a persistent storage (`hostPath` or `storageClass`) from K8S for HAProxy Pods  | `true`                                      |
-| `haproxy.persistence.hostPath` | Sets datadir path on K8S node for all HAProxy Pods. Available only when `pxc.persistence.enabled: true` |                             |
-| `haproxy.persistence.storageClass` | Sets K8S storageClass name for all HAProxy Pods PVC. Available only when `pxc.persistence.enabled: true` | `-`                      |
+| `haproxy.persistence.enabled` | Requests a persistent storage (`hostPath` or `storageClass`) from K8S for HAProxy Pods  | `false`                                      |
+| `haproxy.persistence.hostPath` | Sets datadir path on K8S node for all HAProxy Pods. Available only when `haproxy.persistence.enabled: true` |                             |
+| `haproxy.persistence.storageClass` | Sets K8S storageClass name for all HAProxy Pods PVC. Available only when `haproxy.persistence.enabled: true` | `-`                      |
 | `haproxy.persistence.accessMode` | Sets K8S persistent storage access policy for all HAProxy Pods | `ReadWriteOnce`                      |
 | `haproxy.persistence.size` | Sets K8S persistent storage size for all HAProxy Pods | `8Gi`                      |
```

Comment:
I believe these options added by mistake, haproxy is stateless and shouldn't have persistence options